molecule/plain-rhel/molecule.yml

```diff
@@ -107,6 +107,9 @@ provisioner:
         kafka_connect_confluent_hub_plugins:
           - jcustenborder/kafka-connect-spooldir:2.0.43
 
+        kafka_connect_plugins_remote:
```

Comment:
Can we please check with connect team if this URL would be stable.

Reply:
thanks @nsharma-git , yup checking with connect team https://confluent.slack.com/archives/C6TLCLWKU/p1672727054993709

Reply:
Thanks. We will update the link once we hear back. I don't think we should keep this PR waiting for that.